dev/benchmarks/platform_views_layout/pubspec.yaml

```diff
@@ -9,7 +9,7 @@ dependencies:
     sdk: flutter
   flutter_driver:
     sdk: flutter
-
+  google_mobile_ads: ^4.0.0
```

Comment:
To land this you'll eventually need to run update-packages --force-upgrade, but don't do that now

Reply:
How do the mechanics of that work, will that change this to a specific pin version?

Reply:
You will not be able to check in something that uses a caret today. flutter update-packages will pin it to the latest version that it can.

Reply:
`flutter update-packages --force-upgrade` doesn't change the yaml automatically. I guess it's because 4.0.0 is already the newest version. I will simply remove the caret.

Reply:
You have to remove the caret and then re-run the command